acceptance/config/beaker/local/el6/1host.cfg

```diff
@@ -14,7 +14,7 @@ HOSTS:
     group: puppet
```

Comment:
Can we just remove this from these files? It seems like if everything is set correctly, we shouldn't need to have `puppetpath` etc. set in the hostconfig, because they should be set through beaker's other option-setting capabilities.